<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizWindow</class>
 <widget class="QMainWindow" name="QuizWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>573</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>573</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>なんでもクイズ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>正解数</string>
    </property>
   </widget>
   <widget class="QLabel" name="Correct_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Question_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>第1問</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Choice_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>③</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Choice_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>④</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Choice_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>②</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Choice_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>①</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Yes_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>80</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>○</string>
    </property>
   </widget>
   <widget class="QPushButton" name="No_Button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>80</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="QuizTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>なんでもクイズ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="QuizCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeQuiz">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>変更</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>54</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>問題系統</string>
    </property>
   </widget>
   <widget class="QLabel" name="Quiz_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>問題系統名(デフォ1番上)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>開始</string>
    </property>
   </widget>
   <widget class="QLabel" name="QuestionT_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>問題文</string>
    </property>
   </widget>
   <widget class="QLabel" name="Choice1_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>選択肢１</string>
    </property>
   </widget>
   <widget class="QLabel" name="Choice2_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>選択肢２</string>
    </property>
   </widget>
   <widget class="QLabel" name="Choice3_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>選択肢３</string>
    </property>
   </widget>
   <widget class="QLabel" name="Choice4_label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>選択肢４</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OkButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>決定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>選択中</string>
    </property>
   </widget>
   <widget class="QLabel" name="NowChoice_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>無</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>150</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>次の問題へ</string>
    </property>
   </widget>
   <widget class="QLabel" name="CorrectLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string>正解</string>
    </property>
   </widget>
   <widget class="QLabel" name="CorrectLabel_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ○</string>
    </property>
   </widget>
   <widget class="QLabel" name="NOTLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不正解</string>
    </property>
   </widget>
   <widget class="QLabel" name="NOTLabel_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ×</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="ReGetAction"/>
   </widget>
   <widget class="QMenu" name="OtherMenu">
    <property name="title">
     <string>その他</string>
    </property>
    <addaction name="UpdateAction"/>
    <addaction name="HelpAction"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="OtherMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="UpdateAction">
   <property name="text">
    <string>更新確認</string>
   </property>
  </action>
  <action name="HelpAction">
   <property name="text">
    <string>ヘルプ</string>
   </property>
  </action>
  <action name="ReGetAction">
   <property name="text">
    <string>再読込</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
